ch6/exercise6-1/intset/intset_test.go

```diff
@@ -0,0 +1,69 @@
+package intset
+
+import (
+	"testing"
+)
+
+func TestLen(t *testing.T) {
+	x := &IntSet{}
+	x.Add(127)
+	x.Add(4091)
```

Comment:
Why 4091 here?  In https://github.com/quinqu/tgpl/pull/17 you tested with all small ints ( <=54).  Should one or the other be adjusted?

Reply:
No reason, I will check this out.